app/support/doxxer.rb

```diff
@@ -3,7 +3,11 @@ class Doxxer
 
   def self.example_for(model_klass)
     model = ActiveModel::SerializableResource.new(model_klass.new)
-    JSON.pretty_generate(model.serializable_hash)
+    model_hash = model.serializable_hash
+    [
+      "# Example response JSON",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.